.ci/run.sh

```diff
@@ -9,6 +9,38 @@
 
 set -e
 
+check_log_files()
+{
+	make log-parser
+
+	for component in \
+		kata-proxy \
+		kata-shim \
+		"${RUNTIME}"
+	do
+		file="${component}.log"
+		args="--no-pager -q -o cat -a -t \"${component}\""
+
+		cmd="sudo journalctl ${args} > ${file}"
+		eval "$cmd"
+	done
+
+	dir="."
```

Comment:
just a note: the `dir` feels strangely redundant (just `*.log` would match in `.` anyhow, no?). And, I'd probably have put a `$(pwd)` maybe, but that is probably just personal preference.

Reply:
Fixed.